nodeup/pkg/model/context.go

```diff
@@ -408,6 +408,15 @@ func (c *NodeupModelContext) KubectlPath() string {
 
 // BuildCertificatePairTask creates the tasks to create the certificate and private key files.
 func (c *NodeupModelContext) BuildCertificatePairTask(ctx *fi.ModelBuilderContext, name, path, filename string, owner *string, beforeServices []string) error {
+	return c.buildCertificatePairTask(ctx, name, path, filename, owner, beforeServices, true)
+}
+
+// BuildPrivateKeyTask creates the tasks to create the certificate and private key files.
```

Comment:
This one builds only the private key, doesn't it?
```suggestion
// BuildPrivateKeyTask builds a task to create the private key file.
```